tests/ui/README.md

```diff
@@ -0,0 +1,64 @@
+## CodaLab UI Tests
+
+We test the CodaLab UI by using [Selenium](https://pypi.org/project/selenium/#history) and 
+[diffimg](https://pypi.org/project/diffimg/#history) for our screenshots comparison tool. Currently, our front end 
+automation only runs in Chrome and Firefox. We hope to support more browsers in the future. 
+
+### Setting up the tests on your local machine
+
+#### One-time setup
+
+1. If you haven't already, install the Python dependencies specified in `requirements.txt`, located in the root of the 
+repository, by running `pip install -r requirements.txt`.
+2. Next, install [ChromeDriver](https://chromedriver.chromium.org/getting-started) (for Chrome) and 
```

Comment:
Can we just ask them to run `./scripts/test-setup.py` instead?

Reply:
There are even easier ways to install them depending on your local machine. `test-setup.py` is mainly to get it to work on Travis. I guess I can mention both ways.

Reply:
@epicfaace I actually updated the `test-setup.py` so local installation is a bit easier and still compatible with Travis. I updated the doc too.